ConsumerApi.Sdk/Endpoints/SyncRuns/Types/StartSyncRunRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Backbone.ConsumerApi.Sdk.Endpoints.SyncRuns.Types;
+
+public class StartSyncRunRequest
+{
+    public string? Type { get; set; }
```

Comment:
This can be required and not nullable. It's one of the backwards compatibilities we will remove this sprint.

Reply:
👍